src/main/java/nextstep/ladder/view/ResultView.java

```diff
@@ -29,4 +32,31 @@ private static void printPlayers(Players players) {
     private static void printLines(Lines lines) {
         lines.getLines().forEach(System.out::println);
     }
+
+    public static void printResults(LadderResults ladderResults) {
+        ladderResults.getLadderResults()
+                .forEach(ladderResult -> System.out.print(ladderResult + " "));
+
+        System.out.println();
+    }
+
+    public static void printGameResult(GameResult gameResult, String playerName) {
+        System.out.println();
+        System.out.println("실행결과");
+
+        if(playerName.equalsIgnoreCase("all")) {
+            gameResult.getGameResults()
+                    .forEach(ResultView::printGameResult);
+        } else {
+            printGameResult(gameResult.getResult(new Player(playerName)));
+        }
```

Comment:
```
 2-2. 그리고 내부적으로 if-else를 사용해서 출력을 하고 있는데.. else를 빼자니 메소드를 분리해야 되서 main 메소드가 더 지저분해 지더라구요.. 이 부분의 개선이 필요해 보이는데 조언 부탁드립니다.
```

위와 같은 질문을 남겨주셨네요, 지금과 같은 구조에서는 지금이 최선일 것 같아요 불필요하게 복잡도가 늘어나거나, 구조의 변경이 필요할 것 같아서요

else 구문만을 제거하고 싶으신거라면 아래와 같이는 처리할 수 있을 것 같은데 return이 거슬리네요 ㅠㅠ

```suggestion
        if(playerName.equalsIgnoreCase("all")) {
            gameResult.getGameResults()
                    .forEach(ResultView::printGameResult);
            return;
        }
        gameResult.getResult(new Player(playerName));
```

직접 출력하는게 아니라 GameResult와 playerName를 넘겨받아서 출력할 값(String)을 StringBuilder등을 통해 생성해주는 메소드가 있으면 조금은 더 깔끔하게 정리가 되지 않을까 싶은데, 결국 "all" 여부를 체크하는 부분은 필요해서 크게 코드가 깔끔해질 것 같진 않네요~ 
어떤 방법이 좋을 지 한번 고민해보세요! (추가로 다른 private 메소드를 정의하더라도 해당 ResultView 클래스의 메소드가 전부 static이라 private 메소드 또한 static으로 정의되어야 하겠네요)

Reply:
말씀해주신 것처럼 all 여부를 체크하는 부분이 필요해서 코드가 깔끔해지지 않더라구요..ㅠㅠ
이 부분은 더 고민해보도록 하겠습니다!